module/platform/Webots/src/Webots.cpp

```diff
@@ -792,8 +792,13 @@ namespace module::platform {
             image->format         = fourcc("BGR3");  // Change to "JPEG" when webots compression is implemented
             image->data           = camera.image;
 
-            image->id        = camera_context[camera.name].id;
-            image->timestamp = NUClear::clock::time_point(std::chrono::nanoseconds(sensor_measurements.time));
+            image->id = camera_context[camera.name].id;
+            // TODO unhack
+            // Webots clock and custom clock get out of sync very quickly. HeadBehaviour requires accurate timestamps
+            // on images to ensure it doesn't get stuck fixated on the ball. Safer at this point to save the timestamp
+            // using NUClear clock, however the unsync issue should be investigated
+            image->timestamp = NUClear::clock::now();
```

Comment:
Setting it to `NUClear::clock::now()` should actually be preferred to what we had before anyway
```suggestion
            image->timestamp = NUClear::clock::now();
```